components/ai_chat/core/browser/engine/engine_consumer_llama.cc

```diff
@@ -140,8 +146,10 @@ std::string BuildLlamaSubsequentSequence(
     return base::StrCat({kLlama2Bos, user_message});
   }
 
+  std::string assistant_message = base::StrCat(
```

Comment:
this is a bit confusing to me - it seems like we also pass `kMixtralAssistantTag` in sometimes as the explicit mixtral seed, this seems redundant? is this basically to cover the conversation history case?

Reply:
Yes. It's mainly to maintain and reinforce the consistency of User/Assistant turns in the conversation. Also thought it might help the model further to distinguish the roles, in the mindset of stricter/more explicit prompting for quantised models.

Reply:
makes sense to keep the history consistent, I just found this code a bit confusing - thanks for explaining!

Reply:
makes sense to keep the history consistent, I just found this code a bit confusing - thanks for explaining!